<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="list_reservas">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background:rgba(0,0,0,0);
	border-radius: 25% 25%;
}

QPushButton:hover{
    background:rgba(0,0,0,0.3);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_imprimir">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background:rgba(0,0,0,0);
	border-radius: 25% 25%;
}

QPushButton:hover{
    background:rgba(0,0,0,0.3);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_actualizar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background:rgba(0,0,0,0);
	border-radius: 25% 25%;
}

QPushButton:hover{
    background:rgba(0,0,0,0.3);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/img_reserva.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>29</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background:rgba(0,0,0,0);
	border-radius: 25% 25%;
}

QPushButton:hover{
    background:rgba(0,0,0,0.3);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background:rgba(0,0,0,0);
	border-radius: 25% 25%;
}

QPushButton:hover{
    background:rgba(0,0,0,0.3);
}</string>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <zorder>imgfondo</zorder>
  <zorder>list_reservas</zorder>
  <zorder>btn_cancelar</zorder>
  <zorder>btn_imprimir</zorder>
  <zorder>btn_actualizar</zorder>
  <zorder>btn_close</zorder>
  <zorder>btn_regresar</zorder>
 </widget>
 <resources>
  <include location="img_rc.qrc"/>
 </resources>
 <connections/>
</ui>
